opengrok-indexer/src/main/java/org/opengrok/indexer/history/Annotation.java

```diff
@@ -91,6 +92,15 @@ public String getRevision(int line) {
         return ret;
     }
 
+    @VisibleForTesting
```

Comment:
If the only purpose is to use it only in tests, then maybe `@TestOnly` is a better choice.